src/components/commanderOverview/CommanderDetails.tsx

```diff
@@ -42,6 +48,18 @@ export const CommanderDetails = React.memo(function CommanderDetails() {
         return <Loading text="" />;
     }
 
+    let matchesArray: Match[] = matches;
+    if (searchInput.length > 0 && matches) {
+        matchesArray = matches.filter((match: Match) => {
+            for (let i = 0; i < match.players.length; i++) {

```

Comment:
nit: in this instance, if you aren't using the index, let's use a "for of" loop

Reply:
I was looking for how to do that, thanks